.gitignore

```diff
@@ -46,3 +46,7 @@ source/scpp/extra/DVMChecks.cpp
 
 # Agora log directory when Agora is started up with config.example.yml
 log/
+
+# Infrastructure project containing GeoIp
+infrastructure/
+
```

Comment:
because during development people will need to check out infrastructure(because it contains the mmdb file), and without checking out, 1 test would fail

Reply:
That's why I mentioned that this should soft-fail. Otherwise we're locking out contributors which do not have access to the `infrastructure` repo.

Reply:
we could have also just created a public repo with that one file, so no-one is 'locked-out' and the test can still run